CommonTools/RecoAlgos/interface/RecoTrackSelectorBase.h

```diff
@@ -19,6 +19,8 @@ class RecoTrackSelectorBase {
     ptMin_(cfg.getParameter<double>("ptMin")),
     minRapidity_(cfg.getParameter<double>("minRapidity")),
     maxRapidity_(cfg.getParameter<double>("maxRapidity")),
+    meanPhi_((cfg.getParameter<double>("minPhi")+cfg.getParameter<double>("maxPhi"))/2.),
+    rangePhi_((cfg.getParameter<double>("maxPhi")-cfg.getParameter<double>("minPhi"))/2.),
```

Comment:
@fwyzard Indeed those are still possible, but the code should deal with them properly (thanks to `reco::deltaPhi()` reducing the range of the difference wrt. `meanPhi` to `-π..π`).